ee/tabby-schema/src/schema/repository/third_party.rs

```diff
@@ -0,0 +1,65 @@
+use async_trait::async_trait;
+use chrono::{DateTime, Utc};
+use juniper::{GraphQLObject, ID};
+
+use crate::{integration::IntegrationKind, juniper::relay::NodeType, schema::Result, Context};
+
+#[derive(GraphQLObject)]
+#[graphql(context = Context)]
+pub struct ProvidedRepository {
+    pub id: ID,
+    pub integration_id: ID,
```

Comment:
```suggestion
    pub integration_access_token_id: ID,
```

Can change the name after changing the table name.